integration/spark/app/integrations/container/pysparkRddToCsvCompleteEvent.json

```diff
@@ -10,6 +10,6 @@
   "inputs": [ ],
   "outputs": [ {
     "namespace": "file",
-    "name": "file:/tmp/test_data/rdd_to_csv_output"
+    "name": "/tmp/test_data/rdd_to_csv_output"
```

Comment:
Maybe we should review those files to see if there's any other bug encoded in the check